test/haz3ltest.re

```diff
@@ -4,6 +4,11 @@ let (suite, _) =
   run_and_report(
     ~and_exit=false,
     "Dynamics",
-    [("Elaboration", Test_Elaboration.elaboration_tests)],
+    [
+      ("Elaboration", Test_Elaboration.elaboration_tests),
+      ("Type Assignment", Test_TypeAssignment.type_assignment_tests),
+      ("Random tests", Test_TypeAssignment.random_tests),
```

Comment:
We should still mark these as Type Assignment tests. What do you think about `TypeAssignment.random` or `TypeAssignment.property_tests`  for the name. We still don't have a format for doing hierarchies of tests. Dot separated matching the module structure makes sense to me.